Library/Homebrew/compat/early/download_strategy.rb

```diff
@@ -17,7 +17,7 @@ def _fetch(*args, **options)
       rescue ArgumentError => e
         raise unless e.message.include?("timeout")
 
-        odeprecated "`def _fetch` in a subclass of `CurlDownloadStrategy`"
+        odisabled "`def _fetch` in a subclass of `CurlDownloadStrategy`"
```

Comment:
This was only deprecated a week ago.

Reply:
@reitermarkus Ok. I'll open a PR to address. In future can you add a comment to any `odeprecated` that shouldn't become `odisabled` on the next release?

Reply:
https://github.com/Homebrew/brew/pull/11089

Reply:
No. Most deprecations are only initially added on a new major/minor release.